source/includes/v2/_dataset.md

```diff
@@ -70,177 +78,198 @@ curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets/d
 ```
 
-> Returns an array with one element
+> API reponse
 
 ```json
 {
-    "aggregations": [
-        {
-            "count": 50335
+    "records": [
+        {"record":
+            {"fields":
+                {
+                    "count": 50335
+                }
+            }
         }
     ]
 }
 ```
 
-> Aggregation query with a single group_by
+> Grouping records by a field name (here, `country_code`) and perform an analytic
 
 ```shell
 # Retrieve the total number of cities with more than 5,000 inhabitants, the country code, and the total population for each country code
-curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets/doc-geonames-cities-5000/aggregates?select=count(*) as num_cities,country_code,sum(population) as sum_population&group_by=country_code'
+curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets/doc-geonames-cities-5000/records?select=count(*) as num_cities,country_code,sum(population) as sum_population&group_by=country_code'
 ```
 
-> Returns an array with an object for each `feature` containing the feature's name and number of datasets
+> API reponse
 
 ```json
 {
-    "aggregations": [
+    "records": [
         /* ... */
-        {
-            "num_cities": 1982,
-            "country_code": "FR",
-            "sum_population": 39549205
+        {"record":
+            {"fields":
+                {
+                    "num_cities": 1982,
+                    "country_code": "FR",
+                    "sum_population": 39549205
+                }
+            }
         },
         /* ... */
-        {
-            "num_cities": 7197,
-            "country_code": "US",
-            "sum_population": 243520909
+        {"record":
+            {"fields":
+                {
+                    "num_cities": 7197,
+                    "country_code": "US",
+                    "sum_population": 243520909
+                }
+            }
         },
         /* ... */
-        {
-            "num_cities": 43,
-            "country_code": "ZW",
-            "sum_population": 4043922
+        {"record":
+            {"fields":
+                {
+                    "num_cities": 43,
+                    "country_code": "ZW",
+                    "sum_population": 4043922
+                }
+            }
         }
     ]
 }
 ```
 
-
-> Invalid aggregation with a selected field not present in group_by
+> Invalid analytic on grouped records because no grouping is asked
 
 ```shell
-curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets/doc-geonames-cities-5000/aggregates?select=country_code'
+curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets/doc-geonames-cities-5000/records?select=country_code'
 ```
 
-> Valid aggregation with an aggregation function
+> Valid analytic (here, a `sum`) on grouped records, with an aggregation function
```

Comment:
analytics

Reply:
I suggest "analytics", "valid `sum` analytic function" or "`sum` computation" on grouped record... based on what seems to be more explicit to you.
Open to discussion. (same thing for the other occurrences of "analytic", which is not a noun as pointed out by @mhugo.